kairos-server/src/main.rs

```diff
@@ -1,21 +1,44 @@
-use std::net::SocketAddr;
+use lazy_static::lazy_static;
 
-use dotenvy::dotenv;
-use kairos_server::{config::ServerConfig, state::BatchState};
+use kairos_server::{config::Settings, state::BatchState};
+
+// Globally accessible config
+lazy_static! {
+    static ref CONFIG: Settings = Settings::new();
+}
 
 #[tokio::main]
 async fn main() {
-    tracing_subscriber::fmt::init();
-    dotenv().ok();
-
-    let config = ServerConfig::from_env()
-        .unwrap_or_else(|e| panic!("Failed to parse server config from environment: {}", e));
+    CONFIG.initialize_logger();
```

Comment:
Semantically it is not the config that should initialize logging. It's the server that should initialize logging given a configuration

Reply:
I don't mind moving the logging configuration to main.rs or out of config.rs. Do you have a suggestion?